### DISCUSSION TITLE
Qobra connector

### DISCUSSION THREAD

**Discussion by Tanguy-la-moule** [airbytehq/airbyte#45894] [New Connector Request]
Hello, I'm CTO at Qobra and we often have use cases where customers want to connect Qobra's data to their datalake. For now they go through a custom connection to our public apis, but I'd be available to chat for a regular connection.
The link to our [documentation](https://docs.qobra.co)

**Comment by aazam-gh** [comment]
@Tanguy-la-moule I am interested in building this connector. Do you have a sandbox account I can use for developing the integration?

[a comment by an automated account is not shown]
